docs/static/releasenotes.asciidoc

```diff
@@ -13,6 +14,78 @@ This section summarizes the changes in the following releases:
+   **** DLQ writer interface provided to Java plugins.
+   *** New built-in Java plugins:
+   **** A Java-based generator input for testing with the same capabilities as the Ruby generator input as well as an optional eps option to generate events at a given event-per-second rate for situations where as-fast-as-possible event generation is too much.
+   **** Also includes a jdots codec that mirrors the Ruby dots codec.
+   **** Java-based sink output that discards any events received. Analogous to the Ruby null plugin though much faster.
+   *** Float, URI, and password config types, each of which provide validation for the supplied config value.
+
+
+* Documentation: Add details about Elastic Search dependency https://github.com/elastic/logstash/pull/10852[#10852]
+* Fixed parsing of boolean options provided to Java plugins https://github.com/elastic/logstash/pull/10848[#10848]
+* Field reference: handle illegal field references in converted maps https://github.com/elastic/logstash/pull/10839[#10839]
+* Fixes unit test failures on some runs of ConfigCompilerTest::testComplexConfigToPipelineIR https://github.com/elastic/logstash/pull/10837[#10837]
+* Documentation: Value of start_timestamp must to be quoted https://github.com/elastic/logstash/pull/10836[#10836]
+* Build: name rpm/deb oss packages as logstash-oss https://github.com/elastic/logstash/pull/10833[#10833]
+* LIR support for octal literals in pipeline definitions https://github.com/elastic/logstash/pull/10828[#10828]
+* Merge config values in LIR https://github.com/elastic/logstash/pull/10832[#10832]
+* Build: generate tarballs for docker images https://github.com/elastic/logstash/pull/10819[#10819]
+* Documentation: Document copy semantics of QueueWriter::push method https://github.com/elastic/logstash/pull/10808[#10808]
+* Default stack trace size for hot threads to 50 and make it configurable https://github.com/elastic/logstash/pull/10793[#10793]
+* Include G1 in JVM heap metrics https://github.com/elastic/logstash/pull/10784[#10784]
+* Expose Metrics API to Java plugins https://github.com/elastic/logstash/pull/10761[#10761]
+* Documentation: Clarify behavior of ensure_delivery flag https://github.com/elastic/logstash/pull/10754[#10754]
+* Fix JRuby resolv.rb leak https://github.com/elastic/logstash/pull/10741[#10741]
+* Add LogStash::PluginMetadata for simple key/value plugin metadata https://github.com/elastic/logstash/pull/10691[#10691]
+* Fix default codec and buffer handling in Java stdout output https://github.com/elastic/logstash/pull/10673[#10673]
+* Collect and expose codec metrics https://github.com/elastic/logstash/pull/10614[#10614]
+* Enhance `GET /` API for Metricbeat Merge config values in LIR https://github.com/elastic/logstash/pull/10589[#10589]
+
+
+==== Plugins
+
+*Es bulk Codec*
+
+* Fixed deeplink to Elasticsearch Reference [#18](https://github.com/logstash-plugins/logstash-codec-es_bulk/pull/18)
+
```

Comment:
The plugin items use MD instead of asciidoc formatting. 
Instead of `[#18](https://github.com/logstash-plugins/logstash-codec-es_bulk/pull/18)`, format should be `https://github.com/logstash-plugins/logstash-codec-es_bulk/pull/18 [#18]`, for example.  (This comment applies for all plugin links.)